### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by risyasin**

### CODE REVIEW

File: README-tr.md [hunk 1]
```diff
@@ -0,0 +1,665 @@
+## jQuery'e İhtiyacınız Yok  [![Build Status](https://travis-ci.org/oneuijs/You-Dont-Need-jQuery.svg)](https://travis-ci.org/oneuijs/You-Dont-Need-jQuery)
+
+Önyüz ortamları bugünlerde çok hızlı gelişiyor, öyle ki modern tarayıcılar DOM/DOM APİ'lere ait önemli gereklilikleri çoktan yerine getirdiler. DOM işleme ve olaylar için, en baştan jQuery ögrenmemize gerek kalmadı. Bu arada, üstünlükleri ile jQuery'i önemsizleştiren ve doğrudan DOM değişikliklerinin bir Anti-pattern olduğunu gösteren, React, Angular ve Vue gibi gelişmiş önyüz kütüphanelerine ayrıca teşekkür ederiz. Bu proje, IE10+ desteği ile coğunluğu jQuery yöntemlerine alternatif olan yerleşik uygulamaları içerir. 
+
+## İçerik Tablosu
+
+1. [Sorgu seçiciler](#query-selector)
```
Comment (camsong) [inline README-tr.md]: this anchor link is broken, can you also fix other anchors.

**Comment by camsong** [comment]
you can add a link under https://github.com/oneuijs/You-Dont-Need-jQuery#translations

**Comment by risyasin** [comment]
I always had a problem with conflicting turkish uppercase letter İ. (as in String.toLocaleLowerCase() situation, we had to check it twice) obviously i forgot it again.

**Comment by camsong** [comment]
@risyasin Thanks for the contribution, though I don't understand Turkish, I believe you.
please help us to keep this up to date.

**Comment by risyasin** [comment]
Of course i will keep it up to date. and i would like to thanks for everyone here.

**Pull request merged**